ui/src/common/components/CitationSummaryGraph/CitationSummaryGraph.jsx

```diff
@@ -37,12 +45,102 @@ const xValueToLabel = {
   '250.0-500.0': '250-499',
   '500.0-*': '500+',
 };
+const rangeToQuery = new BiMap({
```

Comment:
so many `map`s in this file, would it be possible to push some logic to backend actually, so that we don't have hardcoded stuff here. @drjova ?

Reply:
also using `xValue`  instead of `range` would be better.

Reply:
we can pass custom keys for each bucket in the aggregation with the `key` key: see second example in https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-range-aggregation.html#_keyed_response_7